### PR TITLE
some borg appearance stuff

### PR DESCRIPTION
**Pull request by Intigracy**
adds new borg sprites http://i.imgur.com/xHRlDR7.gifv
alphabetizes borg and mommi sprite choosing
you get 8 tries to pick your borg sprite, up from 6

mentioning if anyone wants to be a hero and fix these:
#3595
#8254

### CODE REVIEW

**Comment by Exxion** [comment]
Why not just remove the limit on tries?

**Comment by Shadowmech88** [comment]
Why not make the menu for borg sprite picking similar to the RPD's menu, and allow the players to see a preview of each sprite inside the menu?

**Comment by Intigracy** [comment]
The limit is to prevent infinite looping.

@Shadowmech88 WYCI

File: html/changelogs/Intieyebot.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+author: Intigracy
+delete-after: True
+changes: 
+- rscadd: "New borg sprites, meet the eyebot."
+- tweak: "The list where you choose your borg sprite is now alphabetized. Did the MoMMi one too while I was at it."
+- tweak: "You now get 8 tries to pick your borg sprite, up from 8. This system will hopefully be replaced by a menu where you see previews of the borgs."
```
Comment (Exxion) [inline html/changelogs/Intieyebot.yml]: > 8 tries, up from 8

honk

**Comment by gbasood** [comment]
Not a fan of the implementation of the fix, or the sprites, which are pretty lackluster.

**Comment by Intigracy** [comment]
I'm not coding the borg preview, so people are stuck with 6 tries until WYCI since temporary fixes are so evil.

**Pull request closed without merging**